CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
+# v0.1.2
+
+* fam: updated the macro that generates implementions of FamStruct to
```

Comment:
Even though this is a tiny change, I think we might need to use 0.2.0 instead because it is a breaking change.